src/fixtures/userFakeTestData.js

```diff
@@ -0,0 +1,11 @@
+import faker from 'faker';
+
+const userFakeData = {
+  firstName: faker.name.firstName(),
+  lastName: faker.name.lastName(),
+  email: faker.internet.email(),
+  password: faker.internet.password()
+};
+
+
+export default userFakeData;
```

Comment:
Since this file can contain multiple fixtures for a model, use named export instead of default export.